### ISSUE TITLE
User Interface space partitioning

### ISSUE THREAD

**Issue by esques** [Registereth/reddit_dapp#2]
Generally the design needs to be improved: (1) all the text could be bigger and (2) between each text, inputfield, button etc. we can have way more space. (3) between the steps must be more space. this would help the users a lot to gain better overview.

**Comment by esques** [comment]
My original design was not bad at partitioning space IMHO: https://docs.google.com/document/d/1E0xOHANXiIJZklHK6LH8DvrT70OJLIlc0p34CvJxCJc/edit#heading=h.22om8v2aham8

**Comment by kingcocomango** [comment]
So, to blow up the cards to almost fill the page?

**Comment by esques** [comment]
Good point. We have the opportunity to partake at status.im hackathon with it. So maybe we should make it Mobile friendly, (and Status friendly - but I don't know what this would imply...)

**Comment by kingcocomango** [comment]
Currently it looks fine on small screens, last time I checked. AFAIK status injects web3 the same way metamask does, but im not sure.
